### PR TITLE
Mobview

### PR DESCRIPTION
**Pull request by APJ-9**

### CODE REVIEW

File: learned-things/creating-new-elements.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+document.body.onload = addElement
+
+function addElement(){
```
Comment (donjude-eng) [inline learned-things/creating-new-elements.js:3]: write the function in ES6 format.

**const x = (x, y) => { return x * y };**

File: learned-things/media&js.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+function mediaFun(){
+    if(window.innerWidth > 700){
+        document.body.style.backgroundColor = 'red'
```
Comment (donjude-eng) [inline learned-things/media&js.js:3]: rename this file to media.js or index.js

File: learned-things/media&js.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+        console.log('matches')
+    }
+}
+
+// const x = window.matchMedia('(max-width:700px)')
+// console.log(x)
+window.addEventListener('resize',mediaFun)
```
Comment (donjude-eng) [inline learned-things/media&js.js:10]: are you using this listener?

File: learned-things/testing.js [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+basket.href = 'https://www.etsy.com/in-en/cart?ref=hdr-cart'
+
+basket.style.marginRight = '5px'
+
+
+signIn.style.padding = '8px 15px'
+headerHeart.style.padding = '5px 10px'
+basket.style.padding = '5px 10px'
+
+signIn.style.borderRadius = '50px'
```
Comment (donjude-eng) [inline learned-things/testing.js:71]: use prettier and format all these js documents?

File: learned-things/testing.js [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+searchIcon.style.cursor='pointer'
+
+searchIcon.addEventListener('mouseover', ()=>{
+  searchIcon.style.backgroundColor='#ddd'
+})
+searchIcon.addEventListener('mouseout', removeBtnStyle)
+
+
+addElement(rightSideLogo, signIn)
+addElement(rightSideLogo, headerHeart)
```
Comment (donjude-eng) [inline learned-things/testing.js:159]: Try to pass all the arguments as an object 

example:

function someFunc(arg) {
    alert(arg.foo);
    alert(arg.bar);
}
someFunc({foo: "This", bar: "works!"});

File: script.js [hunk 6]
```diff
@@ -0,0 +1 @@
+document.body.style.backgroundColor = 'white'
```
Comment (donjude-eng) [inline script.js:1]: why this?

File: learned-things/testing.js [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+//Logo Styling
+etsyLogo.style.width = '60px'
+etsyLogo.style.cursor = 'pointer'
+
+//Right-side of the logo
+const rightSideLogo = document.createElement('div')
+rightSideLogo.classList.add('rightsidelogo')
+rightSideLogo.style.display = 'flex'
+rightSideLogo.style.alignItems = 'center'
+
```
Comment (donjude-eng) [inline learned-things/testing.js:37]: @APJ-9 also attach screenshots of all the devices here.

**Review by donjude-eng — CHANGES_REQUESTED** [review]

File: learned-things/testing.js (hunk 7 above)
Comment (APJ-9) [reply learned-things/testing.js:37]: Okay
Screenshot from 2022-09-28 11-41-59
Screenshot from 2022-09-28 11-41-51

Comment (APJ-9) [reply learned-things/testing.js:37]: Uploaded My Clone view and original site view

**Comment by APJ-9** [comment]
My Work
Screenshot from 2022-09-28 18-03-21

Original Site 
Screenshot from 2022-09-28 18-03-15

**Comment by APJ-9** [comment]
Created by me:-
Screenshot from 2022-09-29 18-15-44

Orginal Site
Screenshot from 2022-09-29 18-14-58

**Pull request merged**